virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc

```diff
@@ -21,20 +17,23 @@ include::modules/dynamic-provisioning-change-default-class.adoc[leveloffset=+2]
 include::modules/virt-creating-configmap.adoc[leveloffset=+1]
 endif::[]
 
-include::modules/virt-importing-vm.adoc[leveloffset=+1]
+== Importing a virtual machine
+
+include::modules/virt-importing-vm-wizard.adoc[leveloffset=+2]
 
 See xref:../../../virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc#virt-importing-rhv-vm-wizard-fields_virt-importing-rhv-vm[virtual machine wizard fields] for more information on the wizard fields.
 
-include::modules/virt-importing-vm-cli.adoc[leveloffset=+1]
+include::modules/virt-importing-vm-cli.adoc[leveloffset=+2]
 
 ifeval::["{VirtVersion}" == "2.4"]
 include::modules/virt-canceling-vm-import.adoc[leveloffset=+1]
 endif::[]
 
 include::modules/virt-troubleshooting-vm-import.adoc[leveloffset=+1]
 
-[id="virt-importing-rhv-vm-wizard-fields_{context}"]
+[id="virt-importing-rhv-vm-wizard-fields_virt-importing-rhv-vm"]
```

Comment:
You can take out the `_virt-importing-rhv-vm` part since this section is in an assembly.